### PR TITLE
Add Intel Neural Compressor for PyTorch quantization recipes

### PR DESCRIPTION
**Pull request by ftian1**
This PR is used to add a guide on how to use Intel(R) Neural Compressor to quantize PyTorch model easily with accuracy-driven auto tuning mechanism.

### CODE REVIEW

**Comment by ftian1** [comment]
Reviewer, pls let me know if any change is needed:)

**Comment by yqhu** [comment]
Thanks @ftian1 for the contribution.  A couple of comments:
* for Quantization aware training, there is an extra step, `transforms.Normalize((0.1307,), (0.3081,))`, only for the training dataset. Should the normalization be there for all test datasets as well? 
* some indentations are 4 spaces while others are 2 spaces, and some code blocks have a leading whitespace for all lines.  It would be great if they could be reformatted so the users can just copy&paste and run the code without further cleaning up.

Thanks again.

**Comment by ftian1** [comment]
@yqhu  thanks for comments! I have revised the doc accordingly. pls help check again.

**Comment by ftian1** [comment]
@yqhu Patric, May I have your approval on this PR?

**Review by yqhu — APPROVED** [review]
Thanks @ftian1 .  It looks good.
